File: 7kue/Scrolling Text/solve-1.js
function scrollingText(text) {
  const upperText = text.toUpperCase()
  const arr = []
  for (let i = 0; i < upperText.length; i++) {
    arr.push(upperText.slice(i) + upperText.slice(0, i))
  }
  return arr
}

  
console.log(scrollingText('codewars'))

File: 7kue/Scrolling Text/solve-1.py
def scrolling_text(text):
    upper_text = text.upper()
    arr = []
    for i in range(len(upper_text)):
        arr.append(upper_text[i:] + upper_text[:i])
    return arr

print(
    scrolling_text("codewars")
)  # Output: ['CODEWARS', 'ODEWARSC', 'DEWARSCO', 'EWARSCOD', 'WARSCODE', 'ARSCODEW', 'RSCODEWA', 'SCODEWAR']
